resources/views/pages/settings/profile.twig

```diff
@@ -0,0 +1,151 @@
+                {% if config('enable_dect') %}
+                    {{ f.input(
+                        'dect',
+                        __('settings.profile.dect'),
+                        'text',
+                        {'value': user.contact.dect, 'max_length': 40}
+                    ) }}
+                {% endif %}
+                {{ f.input(
+                    'mobile',
+                    __('settings.profile.mobile'),
+                    'text',
+                    {'value': user.contact.mobile, 'max_length': 40}
+                ) }}
+                {% if config('enable_mobile_show') %}
+                    {{ f.checkbox(
+                        'mobile_show',
+                        __('settings.profile.mobile_show'),
+                        user.settings.mobile_show
+                    ) }}
+                {% endif %}
+                {{ f.input(
+                    'email',
+                    __('settings.profile.email'),
+                    'email',
+                    {'value': user.email, 'max_length': 254, 'required': true, 'entry_required_icon': true}
+                ) }}
+            </div>
+
+            <div class="col-md-12">
+                {{ f.checkbox(
+                    'email_shiftinfo',
+                    __('settings.profile.email_shiftinfo', [config('app_name')]),
+                    user.settings.email_shiftinfo
+                ) }}
+                {{ f.checkbox(
+                    'email_news',
+                    __('settings.profile.email_news'),
+                    user.settings.email_news
+                ) }}
+                {{ f.checkbox(
+                    'email_human',
+                    __('settings.profile.email_by_human_allowed'),
+                    user.settings.email_human
+                ) }}
+                {% if config('enable_goody') %}
+                    {% set privacy_email = config('privacy_email') %}
+                    {% set email_goody_label = __('settings.profile.email_goody') ~
+                        (privacy_email ? ' ' ~ __('settings.profile.privacy', [privacy_email]) : '')
+                    %}
+                    {{ f.checkbox(
+                        'email_goody',
+                        email_goody_label,
+                        user.settings.email_goody,
+                        user.settings.email_goody,
+                        false,
+                        true
+                    ) }}
+                {% endif %}
+            </div>
+
+            <div class="col-md-12">
+                {{ f.select(
+                    'shirt_size',
+                    config('tshirt_sizes'),
+                    __('settings.profile.shirt_size'),
+                    user.personalData.shirt_size
+                ) }}
+            </div>
+
+            <div class="col-md-12">
+                {{ m.info(__('settings.profile.angeltypes.info', [url('/angeltypes')]), true) }}
+                {{ f.submit() }}
```

Comment:
Some separation between the button and the text would be nice here, at the moment its in the same line as the info span

Reply:
@frischler that's still an issue if i see that right